### DISCUSSION TITLE
Page duplications with /~/comments - bad for SEO?

### DISCUSSION THREAD

**Discussion by bfercher** [GitbookIO/community#456] [0. General bugs and questions] [answered]
I noticed that all pages now have duplicates with /~/comments at the end of the URL, like [this one](https://docs.readyplayer.me/ready-player-me/integration-guides/unity/early-access-features/~/comments). It seems to have a pretty bad hit on our SEO. Is there a way to disable that?

**Comment by aikoii** [comment]
Hey @bfercher, just wanted to drop a quick reply to let you know we're looking into it 🙏

**Reply by aikoii** [reply, marked as answer]
Hi @bfercher! This should be fixed 🎉🙏
